cosmovisor/README.md

```diff
@@ -12,16 +14,16 @@ go get github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor
 
 ## Command Line Arguments And Environment Variables
 
-All arguments passed to the `cosmovisor` program will be passed to the current daemon binary (as a subprocess). `cosmovisor` will return `/dev/stdout` and `/dev/stderr` of the subprocess as its own. For this reason, `cosmovisor` cannot accept any command line arguments, nor print anything to output (unless it terminates unexpectedly before executing a binary).
+All arguments passed to `cosmovisor` will be passed to the application binary (as a subprocess). `cosmovisor` will return `/dev/stdout` and `/dev/stderr` of the subprocess as its own. For this reason, `cosmovisor` cannot accept any command-line arguments other than those available to the application binary, nor will it print anything to output other than what is printed by the application binary.
 
 `cosmovisor` reads its configuration from environment variables:
 
-* `DAEMON_HOME` is the location where the `cosmovisor/` directory is kept that contains the upgrade binaries (e.g. `$HOME/.gaiad`, `$HOME/.regend`, `$HOME/.simd`, etc.).
+* `DAEMON_HOME` is the location where the `cosmovisor/` directory is kept that contains the genesis binary, the upgrade binaries, and any additional auxiliary files associated with each binary (e.g. `$HOME/.gaiad`, `$HOME/.regend`, `$HOME/.simd`, etc.).
```

Comment:
I think it would make sense to change the name to "source binary" but this would also need to be changed within cosmovisor because cosmovisor is currently set up to read the binary from the `genesis/bin` directory. Maybe we can update this in cosmovisor and then here in the documentation. Let's double check with @robert-zaremba on our daily sync tomorrow.

Reply:
@robert-zaremba recommended that we keep "genesis binary" and that we instruct node operators to add an "upgrade binary" and point the current link to the "upgrade binary" instead of changing the name "genesis binary". Leaving this as is but will update the upgrade guide.